common/main/ja.xml

```diff
@@ -11994,9 +11994,9 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 			<nameField type="given">英子</nameField>
 			<nameField type="given-informal">えいこ</nameField>
 			<nameField type="given2">ソフィア</nameField>
-			<nameField type="surname-prefix">↑↑↑</nameField>
-			<nameField type="surname-core">内田</nameField>
-			<nameField type="surname2">↑↑↑</nameField>
+			<nameField type="surname-prefix">∅∅∅</nameField>
+			<nameField type="surname-core">∅∅∅</nameField>
+			<nameField type="surname2">内田</nameField>
```

Comment:
the test error must be related to this change

Reply:
Yes, none of the Japanese name patterns use surname2 so that never gets used. Not sure what the intent was here.

Reply:
My oversight; will move to -core